tests/test-war-integration/src/main/resources/logging.properties

```diff
@@ -1,5 +1,5 @@
 com.google.cloud.runtimes.level = FINE
-com.google.cloud.runtimes.handlers = com.google.cloud.logging.LoggingHandler
```

Comment:
I would need further test to be sure of that but the output is not redirected to the `ConsoleHandler` either. So I would suspect that the `LoggingHandler` is correctly assigned but does not send the log to Stackdriver, maybe a problem with GRPC (e.g only the RootHandler is correctly initialized).